src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEEventSymbol.cs

```diff
@@ -20,16 +22,18 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols.Metadata.PE
     /// </summary>
     internal sealed class PEEventSymbol : EventSymbol
     {
+#nullable disable // The compiler cannot figure out that this is assigned in the constructor. https://github.com/dotnet/roslyn/issues/39166
```

Comment:
Tagging @jaredpar for nullable guideline: Should we use `#nullable disable` or `= null!;` for fields that are initialized later.

Reply:
Reported #41273 and applied a workaround in code